src/main/java/com/ericsson/gerrit/plugins/eiffel/events/generators/EiffelEventGenerator.java

```diff
@@ -51,6 +63,41 @@ protected static String createRepoURI(String url, String projectName) {
         }
     }
 
+    protected static String getPreviousEiffelEvent(String linkedEiffelEventType, String projectName, String searchCriteria, File pluginDirectoryPath) {
+        try {
+            EventStorage eventStorage = EventStorageFactory.getEventStorage(pluginDirectoryPath, linkedEiffelEventType);
+            String lastEiffelEvent = getEiffelEventIdFromStorage(eventStorage, projectName, searchCriteria);
+            return lastEiffelEvent;
+        } catch(IllegalArgumentException e) {
+            LOGGER.error("Failed creating link.", e);
```

Comment:
Is this error message correct or something left?

Reply:
Not sure... any idea?

Reply:
Could not get previous eiffel event